### DISCUSSION TITLE
solar optimizer starts the boiler during the night

### DISCUSSION THREAD

**Discussion by jms92100** [jmcollin78/solar_optimizer#34] [Q&A]
![image](https://github.com/jmcollin78/solar_optimizer/assets/29059219/c1bb6789-4c11-437f-8a10-eb760d91eb1a)

The net power consumption is positive ! thats the reason

**Comment by jmcollin78** [comment]
Hello, you should format your configuration file with the <> button. It is not readable as is.

**Comment by jms92100** [comment]
I created a sensor with a negative value when i send power to my supplier. (reversed the initial sensor) 
And thats fine it works the way it should
